crates/torii/migrations/20240314182410_event_model.sql

```diff
@@ -0,0 +1,53 @@
+-- NOTE: sqlite does not support deleteing columns. Workaround is to create new table, copy, and delete old.
+
+CREATE TABLE event_messages (
+    id TEXT NOT NULL PRIMARY KEY,
+    keys TEXT,
+    event_id TEXT NOT NULL,
+    model_names TEXT,
+    created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE INDEX idx_event_messages_keys ON event_messages (keys);
+
+CREATE INDEX idx_event_messages_event_id ON event_messages (event_id);
+
+-- Create new table without model_names column
+CREATE TABLE event_messages_new (
```

Comment:
Mmm I think line 17 - 42 can be deleted now since new `event_messages` table is created here.

Reply:
Ohh but what is it for then? Because that's done for the entities table

Reply:
Ah it's a workaround for deleting columns of existing tables. But since u r creating a new table it's not needed.

https://github.com/dojoengine/dojo/blob/35d789a77156cff98b7c3c43b12372f187cd394d/crates/torii/migrations/20231127235011_entity_model.sql#L1